src/builder/index.js

```diff
@@ -44,6 +46,14 @@ export default function(options) {
         }
       })
 
+      // Copy static assets to build folder
+      if (options.staticAssets !== undefined) {
+        const copySource = options.staticAssets.source
```

Comment:
@MoOx We could have some check with `staticAssets.source`

I'm passing a absolute path in my project https://github.com/thangngoc89/blog/blob/3fd918097603c747a3731b7a1ee088eeee13d6b1/scripts/build.js#L159

Not sure if it could handle relative path